lms/djangoapps/certificates/docs/decisions/003-web-certs.rst

```diff
@@ -0,0 +1,111 @@
+Web (HTML) course certificates
+==============================
+
+Status
+------
+Accepted
+
+Background
+----------
+edX-platform currently has code for both PDF and web (or HTML) course
+certificates. Over time, much of the PDF certificate code has been deprecated
+or disabled and it will likely be removed in a future release.
```

Comment:
> it will likely be removed in a future release
The language here makes me pause, because this doesn't feel like we have made a decision. It feels more like a PSA over a decision record to me. I fully understand _why_ it was written this way but I think the hesitation was still worth mentioning.

My thoughts... 
- Maybe this ADR should have stronger language around the intention of removing support for PDF certificates and we updated the status to `Proposed`? (Until we get the needed clarity on this decision?)
- Should there be a `consequences` section that explains if PDF certificate support is removed that PDF certificates won't be viewable until web certificate support is enabled for in their edX installation (and applicable course-run?).